Lombiq.NodeJs.Extensions/package.json

```diff
@@ -3,45 +3,54 @@
-    "_rm:temp:styles": "cd ../../.. && rimraf $SCSS_TARGET/_temp_",
-    "clean:styles": "SCSS_TARGET=wwwroot/css pnpm _clean:styles",
-    "clean:styles:args": "SCSS_TARGET=$npm_config_target pnpm _clean:styles",
-    "_clean:styles": "cd ../../.. && rimraf \"$SCSS_TARGET/**/*.*\"",
+    "lint:styles": "LINT_TARGET_DIR=../.. SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _lint:styles --silent",
+    "lint:styles:args":
+      "LINT_TARGET_DIR=../.. SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _lint:styles --silent",
+    "_lint:styles": "stylelint \"$LINT_TARGET_DIR/$SCSS_SOURCE/**/*.scss\" || ERRORSTEP=stylelint pnpm _error:styles",
+    "compile:styles": "TARGET_DIR=../../.. SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _compile:styles",
+    "compile:styles:args":
+      "TARGET_DIR=../../.. SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _compile:styles",
+    "_compile:styles":
+      "run-s --silent _sass:styles _autoprefix:styles _minify:styles _rm:temp:styles || ERRORSTEP=compilation pnpm _error:styles",
+    "_sass:styles": "cd $TARGET_DIR && sass $SCSS_SOURCE:$SCSS_TARGET/_temp_ --embed-sources",
+    "_autoprefix:styles":
+      "cd $TARGET_DIR && postcss $SCSS_TARGET/_temp_/**/*.css --base $SCSS_TARGET/_temp_ --dir $SCSS_TARGET --use autoprefixer --map",
+    "_minify:styles":
+      "cd $TARGET_DIR && cleancss --batch --batch-suffix .min \"$SCSS_TARGET/**/*.css\" \"!$SCSS_TARGET/**/*.min.css\" \"!$SCSS_TARGET/_temp_/**/*\"",
+    "_rm:temp:styles": "cd $TARGET_DIR && rimraf $SCSS_TARGET/_temp_",
+    "clean:styles": "TARGET_DIR=../../.. SCSS_TARGET=wwwroot/css pnpm _clean:styles",
+    "clean:styles:args": "TARGET_DIR=../../.. SCSS_TARGET=$npm_config_target pnpm _clean:styles",
+    "_clean:styles": "cd $TARGET_DIR && rimraf \"$SCSS_TARGET/**/*.*\"",
     "watch:styles": "SCSS_SOURCE=Assets/Styles SCSS_TARGET=wwwroot/css pnpm _watch:styles",
     "watch:styles:args": "SCSS_SOURCE=$npm_config_source SCSS_TARGET=$npm_config_target pnpm _watch:styles",
-    "_watch:styles": "nodemon --watch ../../$SCSS_SOURCE --ext scss --exec \"pnpm build:styles:args --source=$SCSS_SOURCE --target=$SCSS_TARGET\"",
-    "_error:styles": "echo \"Error in styles pipeline: $ERRORSTEP failed.\""
+    "_watch:styles":
+      "nodemon --watch ../../$SCSS_SOURCE --ext scss --exec \"pnpm build:styles:args --source=$SCSS_SOURCE --target=$SCSS_TARGET\"",
+    "_error:styles": "echo \"Error in styles pipeline: $ERRORSTEP failed.\"",
+    "--": "-----------------------------------------------------------------------------------------------------------",
+    "__shell_docs__": "https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-shell/tests/shell.test.ts"
   },
   "browserslist": [
-    "last 1 Chrome version",
-    "last 1 Edge version",
-    "last 1 Firefox version",
-    "last 1 IE version",
-    "last 1 iOS version"
+    "defaults"
```

Comment:
@Piedone This seems to be a [much better default](https://github.com/browserslist/browserslist#:~:text=defaults%3A%20browserslist%E2%80%99s%20default%20browsers) than what we had before.

Reply:
Yep.